hyco-https/tests/post.test.js

```diff
@@ -10,18 +10,29 @@ expect(path).toBeDefined();
 expect(keyrule).toBeDefined();
 expect(key).toBeDefined();
 
-// create a large message over 64kb to force a rendezvous connection
-var largeMessage = "";
-for (var i = 1024 * 64; i >= 0; i--) {
-    largeMessage += String.fromCharCode(i % 128);
+var smallMessage = "SmallMessage";
+var exactly64kbMessage = "";
+for (var i = 1024 * 64; i > 0; i--) {
+    exactly64kbMessage += String.fromCharCode(i % 128);
 }
+var over64kbMessage = exactly64kbMessage + String.fromCharCode(0);
 
-function sendAndReceive(requestMsg, responseMsg, done) {
-    var totalRequests = 10; // Total requests to send over the test
```

Comment:
>var totalRequests = 10 [](start = 4, length = 22)

Too many tests now to run multiple iterations. All tests should take 20-25 seconds now.